python/lsst/afw/geom/ellipses/ellipse.cc

```diff
@@ -22,24 +22,36 @@
 
 #include <pybind11/pybind11.h>
 //#include <pybind11/operators.h>
-//#include <pybind11/stl.h>
+#include <pybind11/stl.h>
+
+#include "lsst/afw/geom/ellipses/BaseCore.h"
+#include "lsst/afw/geom/Point.h"
+#include "lsst/afw/geom/ellipses/Ellipse.h"
 
 namespace py = pybind11;
+using namespace pybind11::literals;
```

Comment:
Got it. That's certainly an excellent justification and I agree it should be kept as is, without explanatory comment.